### DISCUSSION TITLE
NoClassDefFoundError: org/eclipse/rdf4j/model/URI

### DISCUSSION THREAD

**Discussion by naturzukunft** [eclipse-rdf4j/rdf4j#3510] [General]
Hi there, i'm a little bit confused. i upgraded to 4.0.0-M1 and get the following error:

```
Caused by: java.lang.NoClassDefFoundError: org/eclipse/rdf4j/model/URI
	at org.eclipse.rdf4j.model.util.Values.<clinit>(Values.java:58)
	at org.linkedopenactors.ap.repository.SimpleRdfTypeRepository.<clinit>(SimpleRdfTypeRepository.java:39)
```
```
class SimpleRdfTypeRepository implements RdfTypeRepository {
	
39:	private static final IRI TEMP_MODEL = Values.iri("http://example.com/tempModel");

```
Fredy

**Comment by barthanssens** [comment]
Hmz, URI is deprecated, so IRI should be used instead... But that should not break the Values.iri method... 
Have you tried `mvn clean` and rebuild you project from scratch ?

**Comment by naturzukunft** [comment]
See https://github.com/eclipse/rdf4j/discussions/4071
there were transitive dependencies from an apache-camel bom
